packages/common-all/src/types/workspace.ts

```diff
@@ -238,6 +238,9 @@ export type DendronConfig = {
    * Configuration for Random Note Lookup Command
    */
   randomNote?: RandomNoteConfig;
+
+  /** Automatically fold frontmatter when opening a new note. False by default. */
+  autoFoldFrontmatter?: boolean;
```

Comment:
awesome job with this. only nitpick is to default this to `true`. when possible, we opt for [sensible defaults](https://github.com/dendronhq/dendron-site/blob/dev/vault/dendron.principles.md#sensible-defaults)